cli.js

```diff
@@ -43,28 +44,40 @@ function sortPackageJsonFile(file, reporter, isCheck) {
   reporter.reportChanged(file)
 }
 
-function sortPackageJsonFiles(patterns, options) {
-  const files = globSync(patterns)
+function sortPackageJsonFiles(patterns, { ignore, ...options }) {
+  performance.mark('start')
+  const files = globSync(patterns, { ignore })
   const reporter = new Reporter(files, options)
   const { isCheck } = options
 
+  performance.mark('sorting')
   for (const file of files) {
     try {
       sortPackageJsonFile(file, reporter, isCheck)
     } catch (error) {
       reporter.reportFailed(file, error)
     }
   }
-
+  performance.mark('finish')
   reporter.printSummary()
+  if (process.env.DEBUG) {
+    console.log([
+      performance.measure('Glob Search', 'start', 'sorting'),
+      performance.measure('Sorting', 'sorting', 'finish'),
+      performance.measure('Total', 'start', 'finish'),
+    ])
+  }
```

Comment:
Can we remove this one please? I'd prefer to avoid shipping debug code. It's easy enough to add in for those interested in debugging.

Reply:
sure